packages/ui/src/components/GlobalStyles.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { createGlobalStyle } from 'styled-components';
+
+export const svgGradients = {
+  gold: 'gold-gradient',
+};
+
+export const Styles = createGlobalStyle`
+* {
+  box-sizing: border-box;
+  margin: 0px;
+  padding: 0px;
+  font-family: 'Poppins';
+}
+
+
+::-webkit-scrollbar {
+  width: 4px;
+}
+
+/* Track */
+::-webkit-scrollbar-track {
+  background-color: transparent;
+  border-radius: 6px;
+}
+
+/* Handle */
+::-webkit-scrollbar-thumb {
+  background-color: #46403C;
+  border-radius: 6px;
+}`;
+
+export const GlobalStyles: React.FC = () => (
+  <>
+    <Styles />
+    <svg xmlns="http://www.w3.org/2000/svg">
```

Comment:
This renders in the app, it should be hidden

Reply:
sorry, what do you mean?

Reply:
if you run the app you can see this rendering at the top which displaces the ui

Reply:
![image](https://github.com/Cypherock/cypherock-cysync/assets/107414753/5d638f3a-a281-4446-bef2-9a8b98fdf6da)

Reply:
ah i see, setting display=none fixes this, sorry about that